src/tg/index.js

```diff
@@ -64,11 +71,11 @@ var init = function(msgCallback) {
             }
 
             if (message.user) {
-                message.text = '<' + message.user + '> ' + message.text;
+                message.text = '<' + message.user + '> ' + irc_to_html_format(message.text);
```

Comment:
Furthermore `<username>` will be interpreted as an XML tag and probably will end up being removed by telegram. We need to escape all occurences of `<` and `>` first.